compilers/esm.js

```diff
@@ -101,7 +101,7 @@ exports.compile = function(load, opts, loader) {
       lowResolutionSourceMap: opts.lowResSourceMaps
     });
 
-    var tree = load.metadata.parseTree || compiler.parse(load.source, load.path);
+    var tree = load.metadata.parseTree || compiler.parse(source, load.path);
```

Comment:
In that case, did you change the wrong one then? You should change this one right? https://github.com/systemjs/builder/pull/509/files#diff-201f308f4e4951a0b727bbb5641b2e3eR149